prov/efa/src/efa_hmem.c

```diff
@@ -44,43 +44,69 @@ static size_t efa_max_eager_msg_size_with_largest_header(struct efa_domain *efa_
 
 	return mtu_size - rxr_pkt_max_hdr_size();
 }
+#else
+static size_t efa_max_eager_msg_size_with_largest_header(struct efa_domain *efa_domain) {
+	return 0;
+}
 #endif
 
+static int efa_hmem_info_init_msg_sizes(struct efa_hmem_info *info, enum fi_hmem_iface iface, struct efa_domain *efa_domain) {
```

Comment:
This function name is not intuitive.. it is actually setting the protocol threshold. I would prefer `efa_hmem_info_init_protocol_thresholds`

Reply:
Agree, thanks